conf/ocsci/ocs-4.3.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 DEPLOYMENT:
-  ocs_registry_image: "quay.io/rhceph-dev/ocs-registry:latest-4.3"
+  default_ocs_registry_image: "quay.io/rhceph-dev/ocs-registry:latest-4.3"
```

Comment:
Can you also change this to point to the `ocs-olm-operator` instead of the deprecated `ocs-registry`?

Reply:
Sure, I haven't noticed this and just changed the name of the parameter for this PR. I changed also this in next commit.